keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -275,21 +280,32 @@ class FasterRCNN(tf.keras.Model):
             (all foreground classes + one background class). By default it uses the rcnn head
             from paper, which is 2 FC layer with 1024 dimension, 1 box regressor and 1
             softmax classifier.
+        prediction_decoder: (Optional)  A `keras.layer` that is responsible for
+            transforming RetinaNet predictions into usable bounding box Tensors.  If
```

Comment:
maybe we should clarify what the model output is before the decoder layer. "transforming RetinaNet predictions" sounds weird in this file.

Reply:
Good catch -- swapped RetinaNet for FasterRCNN and added explanation of box output before decoding.